src/components/Navigation.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import { BrowserRouter as Router, Route, Link } from 'react-router-dom';
+
+import ActorPage from './ActorPage';
+import MoviePage from './MoviePage';
+import MoviesList from './MoviesList';
+
+class Navigation extends Component {
+  render() {
+    return (
+      <div className="App">
+        <Router>
+          <div>

```

Comment:
Do you need this `div` here?

Reply:
It seems necessary. Without this <div> wrapper - react fails with message 

> Invariant Violation: A <Router> may have only one child element

Reply:
But I replaced it with Fragment and everything is fine to.